tests/test_sanity.py

```diff
@@ -874,16 +874,6 @@ def prep():
       wipe()
       self.do([PYTHON, EMCC]) # first run stage
       try_delete(tag_file)
-      config = open(CONFIG_FILE).read()
-      assert '''BINARYEN_ROOT = os.path.expanduser(os.getenv('BINARYEN', ''))''' in config, config # setup created it to be ''
-      print('created config:')
-      print(config)
-      restore_and_set_up()
-      config = open(CONFIG_FILE).read()
-      config = config.replace('BINARYEN_ROOT', '''BINARYEN_ROOT = os.path.expanduser(os.getenv('BINARYEN', '')) # ''')
-      print('modified config:')
-      print(config)
-      open(CONFIG_FILE, 'w').write(config)
```

Comment:
why was this test code removed?

Reply:
As far as I can tell this wasn't doing anything.  It looks like it verifies that BINARYEN_ROOT defaults to empty string.  Then it sets BINARY_ROOT to the empty string.   I'm I missing something?

Reply:
Hmm I think you're right, I can't see what this code does either.